ui-poc/setup_test_data.py

```
@@ -0,0 +1,603 @@
+to simulate real-world scenarios for testing the Splitwiser application.
+
+Features:
+- Creates 5 different user accounts
+- Sets up 3 groups with different member combinations
+- Adds various types of expenses (equal splits, unequal splits, percentages)
+- Creates a complex web of debts and credits between users
+- Includes different scenarios like travel, dining, utilities, etc.
+"""
+
+import requests
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)